scm-ui/ui-webapp/src/repos/containers/Overview.tsx

```diff
@@ -125,13 +134,15 @@ class Overview extends React.Component<Props> {
 
 const mapStateToProps = (state: any, ownProps: Props) => {
   const { match } = ownProps;
+  const me = fetchMe(state);
```

Comment:
Why do we need me here? Why does the repository overview depend on the me resource?